src/MetaMorpho.sol

```diff
@@ -777,11 +791,23 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
             Id id = withdrawQueue[i];
             MarketParams memory marketParams = _marketParams(id);
 
+            uint256 supplyShares = MORPHO.supplyShares(id, address(this));
```

Comment:
before the code was much simpler

Reply:
You mean because we were internalizing half of the code into `_suppliable`?